t/10-jobs.t

```diff
@@ -875,4 +880,41 @@ subtest '"race" between status updates and stale job detection' => sub {
 
 is $t->app->minion->jobs({states => ['failed']})->total, 0, 'No unexpected failed minion background jobs';
 
+subtest 'special cases when restarting job via Minion task' => sub {
+    $ENV{OPENQA_JOB_RESTART_ATTEMPTS} = 2;
+    $ENV{OPENQA_JOB_RESTART_DELAY} = 1;
```

Comment:
I would always put a `local` in front, to avoid accidentally affecting later tests (when they are added in the future).